### ISSUE TITLE
Install AddHTMLMetaAndTitle

### ISSUE THREAD

**Issue by Southparkfan** [miraheze/mw-config#346]
@revi @Reception123 @NDKilla:

The extension in https://github.com/miraheze/mediawiki/commit/db2eab6f711470a3c3d9ed03dc706fa9dfbab804 passed my review and should be enabled on permanentfuturelabwiki and extloadwiki.

Please enable this extension per the usual procedures. 

Thanks,

Southparkfan

**Comment by NDKilla** [comment]
@miraheze/mediawiki-admins may be useful in the future. Planned for later tonight.

**Comment by Southparkfan** [comment]
Installed on extloadwiki, but enabling in prod blocked by https://gerrit.wikimedia.org/r/#/c/268693/.

**Comment by Southparkfan** [comment]
Was already done a while ago.
